src/resolvers.js

```diff
@@ -315,25 +316,18 @@ export class NewInstance {
 }
 
 export function getDecoratorDependencies(target, name) {
```

Comment:
Can we remove the `name` parameter here and then remove it from the places it's called below?